.gitignore

```diff
@@ -0,0 +1,4 @@
+.*
+*~
+"#"*"#"
+!.gitignore
```

Comment:
There are only 3 files in this repository and none of them start with `.`
What problem does this file solve?

Reply:
I code using emacs and this saves me from cleaning editor tmp files.
I think this is some kind of common practice but I can also remove it (not really a issue for me), would you like me to remove it?

Reply:
Don't think I am the right person to make this call, just haven't seen those ignores in elisp repositories much.